packages/blocks/src/_common/adapters/markdown.unit.spec.ts

```diff
@@ -1003,6 +1003,213 @@ hhh
     });
     expect(target.file).toBe(markdown);
   });
+
+  test('table', async () => {
+    const blockSnapshot: BlockSnapshot = {
+      type: 'block',
+      id: 'block:8Wb7CSJ9Qe',
+      flavour: 'affine:database',
+      props: {
+        cells: {
+          'block:P_-Wg7Rg9O': {
+            'block:qyo8q9VPWU': {
+              columnId: 'block:qyo8q9VPWU',
+              value: 'TKip9uc7Yx',
+            },
+            'block:5cglrBmAr3': {
+              columnId: 'block:5cglrBmAr3',
+              value: 1702598400000,
+            },
+            'block:8Fa0JQe7WY': {
+              columnId: 'block:8Fa0JQe7WY',
+              value: 1,
+            },
+            'block:5ej6StPuF_': {
+              columnId: 'block:5ej6StPuF_',
+              value: 65,
+            },
+            'block:DPhZ6JBziD': {
+              columnId: 'block:DPhZ6JBziD',
+              value: ['-2_QD3GZT1', '73UrEZWaKk'],
+            },
+            'block:O8dpIDiP7-': {
+              columnId: 'block:O8dpIDiP7-',
+              value: {
+                _yText: 'test1',
```

Comment:
I'm sorry but could you please merge or rebase the change of latest commit?
We have a fix for it. https://github.com/toeverything/blocksuite/commit/b7782ebf4ee3984ca4f9bb32cdbe2c4828d2bd73

Reply:
fixed

Reply:
fixed